### ISSUE TITLE
Minimum dataset size for intelligible speech synthesis?

### ISSUE THREAD

**Issue by sharathadavanne** [NVIDIA/tacotron2#251]
In this work from [Google](https://google.github.io/tacotron/publications/semisupervised/index.html) they claim that the  baselineTacotron2 can produce intelligible speech with just 1.5 shards (36 minutes) of the dataset. Has anyone done a similar experiment with this (NVIDIA's) implementation of Tacotron2?

I am asking this question because I was currently working on two different speaker models, one with A) 11 hours and one with B) 6 hours of training data. Each of these has a fairly balanced phone distribution. Model A and B both converged (250k+ iterations) during the training stage producing diagonal attention outputs. But, during inference, model A works well for any input text, but B simply does not synthesize intelligible audio for any input text other than the training and validation text (weird). As a check for overfitting, I tried to synthesize with different checkpoints of model B ( the validation loss curve is a continuously decreasing curve hence no sure shot checkpoint to choose), but the same un-intelligible speech was synthesized with all input text. I am not really sure if it is because of the dataset size? or is it because of some difference in implementation of this repository in comparison to Google's implementation, that introduces a limitation requiring larger dataset size for converging?

Any thoughts or pointers are welcome.

**Comment by rafaelvalle** [comment]
Yes, this is possible by starting from a pre-trained multi-speaker tacotron that was pre-trained with  speaker embeddings on LJS. In this setup, the speaker embedding is learned afresh.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
